website/cue/reference/components/sources/base/host_metrics.cue

```diff
@@ -1,57 +1,6 @@
 package metadata
 
 base: components: sources: host_metrics: configuration: {
-	cgroups: {
```

Comment:
Doh, yeah, can you back these changes to the `cgroups` changes back out. You ran into https://github.com/vectordotdev/vector/issues/15193 . 😓 Sorry for the back-and-forth on this.

Reply:
No worries! That's been reverted.